freud/data.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2010-2019 The Regents of the University of Michigan
+# This file is from the freud project, released under the BSD 3-Clause License.
+
+R"""
+The :class:`freud.data` module provides certain sample data sets and utility
+functions that are useful for testing and examples.
+"""
+
+import numpy as np
+import freud
+
+
+class UnitCell(object):
+    """Class to represent the unit cell of a crystal structure.
+
+    This class represents the unit cell of a crystal structure, which is
+    defined by a lattice and a basis. It provides the basic attributes of the
+    unit cell as well as enabling the generation of systems of points
+    (optionally with some noise) from the unit cell.
+
+    Args:
+        box:
+            A box-like object (see :meth:`~freud.box.Box.from_box`) containing
+            the lattice vectors of the unit cell.
+        basis_positions ((:math:`N_{points}`, 3) :class:`numpy.ndarray`):
+            The basis of the unit cell in fractional coordinates
+            (Default value = ``[[0, 0, 0]]``).
+    """
+    def __init__(self, box, basis_positions=[[0, 0, 0]]):
+        self._box = freud.box.Box.from_box(box)
+        self._basis_positions = basis_positions
+
+    def to_system(self, num_replicas=1, scale=1, sigma_noise=0, seed=None):
+        """Generate a system from the unit cell.
+
+        The box and the positions are expanded by ``scale``, and then Gaussian
+        noise with standard deviation ``sigma_noise`` is added to the
+        positions.  All points are wrapped back into the box before being
+        returned.
+
+        Args:
+            num_replicas (:class:`tuple` or int):
+                If provided as a single number, the number of replicas in all
+                dimensions. If a tuple, the number of times replicated in each
+                dimension. Must be of the form (nx, ny, 1) for 2D boxes
+                (Default value = 1).
+            scale (float):
+                Factor by which scale the unit cell (Default value = 1).
+            sigma_noise (float):
+                The standard deviation of the normal distribution used to add
+                noise to the positions in the system (Default value = 0).
+            seed (int):
+                If provided, used to seed the random noise generation. Not used
+                unless ``sigma_noise`` > 0 (Default value = None).
```

Comment:
I think we use ``:code:`sigma_noise` `` instead of double backticks everywhere else in freud. Also check how we format `None` elsewhere -- I think we also put that in code font.
```suggestion
                unless :code:`sigma_noise` > 0 (Default value = None).
```